package.json

```
@@ -57,6 +57,7 @@
     "decoders": "1.25.5",
     "fp-ts": "2.16.9",
     "io-ts": "2.2.21",
+    "jet-schema": "^1.4.2",
```

Please use exact version

```suggestion
    "jet-schema": "1.4.2",
```